tagstudio/resources/translations/en.json

```
@@ -197,6 +197,7 @@
     "tag.color": "Color",
     "tag.confirm_delete": "Are you sure you want to delete the tag \"{tag_name}\"?",
     "tag.create": "Create Tag",
+    "tag.create_quick": "Create && Add \"{query}\"",
```

Nit: I would rename this to `tag.create_add` to be inline with `open_create_library` and other keys.
(I have added the other required changes as separate comments)
```suggestion
    "tag.create_add": "Create && Add \"{query}\"",
```